docs/support.md

```diff
@@ -1,6 +1,6 @@
 # Support on Apply
 
-There’s one person from Apply (Candidate + ProVendor) on support every week. Next week’s dev is also on hand as the backup support dev.
+There’s one person from Apply (Candidate + Provider) on support every week. Next week’s dev is also on hand as the backup support dev.
```

Comment:
we can get rid of the text in parentheses here, as it reflects a world where Apply was two teams (one for the candidate flow, one for providers and vendor, hence "provendor")